apps/sasquatch/src/main/java/com/microsoft/appcenter/sasquatch/activities/ManagedErrorActivity.java

```diff
@@ -71,17 +71,14 @@ protected void send(View view) {
         } catch (Throwable t) {
             try {
                 Map<String, String> properties = readStringProperties();
-
-                @SuppressWarnings("JavaReflectionMemberAccess")
-                Method method = Crashes.class.getDeclaredMethod("trackException", Throwable.class, Map.class, Iterable.class);
-                method.setAccessible(true);
+                Method method = Crashes.class.getMethod("trackError", Throwable.class, Map.class, Iterable.class);
```

Comment:
Before trackException would work in jCenter with name and properties via reflecting on a private API.

So the accessible trick was already broken after adding attachments (signature changed), now it's even worse by renaming.

Just remove that and we will need jCenter update.